packages/footer/src/scss/_component.scss

```diff
@@ -59,46 +59,31 @@
     }
 
     &__row {
-        display: flex;
-        flex-flow: row wrap;
-        justify-content: space-between;
-        align-items: stretch;
-	    align-content: stretch;
-    }
-
-    &__column {
-        flex: 1 1 100%;
-        padding: core.$space-none (nth(grid.$schema-base, 2) / 2) core.$space-none core.$space-none;
-
-        + .uq-footer__column {
-            flex: 1 1 100%;
-
-            @media #{core.$screen-md-up} {
-                flex: 1 1 25%;
-            }
-        }
-
-        &:last-child {
-            padding: core.$space-none;
-        }
-
+        display: grid;
+        grid: auto-flow / 1fr;
+        gap: 1.5rem;
```

Comment:
this could be increased a little more timmy for medium if possible?

Reply:
Done. Gap increased between columns

Reply:
sorry timmy - few more minor things here... we may as well adjust these now while we are looking at them. Could we tweak the vertical spacing/paddings for the elements when on mobile? More so to evenly space the elements. I can talk you through this if you need more info.

Reply:
Done